spec/services/bulk_api_import/fhir_condition_importer_spec.rb

```diff
@@ -3,8 +3,13 @@
 RSpec.describe BulkApiImport::FhirConditionImporter do
   before { create(:facility) }
   let(:import_user) { ImportUser.find_or_create }
+  let(:org_id) { import_user.organization_id }
   let(:identifier) { SecureRandom.uuid }
-  let(:patient) { build_stubbed(:patient, id: Digest::UUID.uuid_v5(Digest::UUID::DNS_NAMESPACE, identifier)) }
+  let(:patient) do
+    build_stubbed(:patient, id: Digest::UUID.uuid_v5(
+      Digest::UUID::DNS_NAMESPACE + org_id + "patient_business_identifier", identifier
+    ))
+  end
```

Comment:
One weak point in our design is that we must vigilantly ensure that resource creation and resource lookup are in sync. If someone were to change the way we map an ID on resource creation in the future, the subsequent lookups from various importers are going to fail, and this unit test cannot catch this.

The only way to ensure that we are in sync is to write an integration test suite that creates a resource using the API and then subsequently updates it and asserts on the new state. Right now I have left such a test out because it would require us to spin up sidekiq or stub it with a functional equivalent that actually persists data—I felt that this is not yet worth the effort for such a nascent API. Moreover, this issue is slightly mitigated by the fact that ID creation and ID lookup are reusing the same method, so we are protected against the modification of the mapping implementation itself.